releases.yaml

```diff
@@ -0,0 +1,16 @@
+sources:
+  - id: keycloak
+    productName: Keycloak
+    repo: keycloak/keycloak
+    githubReleaseFrom: 23.0.0
```

Comment:
Would rename this to `githubReleaseNotesFrom` - since it was in `GitHubReleaseNotesBuilder` in the past it made sense, but no it seems it's `GitHub Releases` from, and not release notes from.

Reply:
Done